pkg/square/builder.go

```diff
@@ -366,11 +366,6 @@ func (b *Builder) IsEmpty() bool {
 	return b.TxCounter.Size() == 0 && b.PfbCounter.Size() == 0
 }
 
-// TODO: celestia-core should provide this method for `Blob`s
```

Comment:
[Question][Almost unrelated]
Why does core need to know about blobs and namespaces?

Reply:
hmm good question. Basic types like `Blob` and `BlobTx` are defined in celestia-core 

e.g. https://github.com/celestiaorg/celestia-core/blob/d5cd08c4e13a66acc087eda5d43109928e630846/types/block.go#L1039-L1055

So at least for now celestia-core is aware of blobs

Reply:
Thanks. Just checked, apparently, that's only used for tests

Reply:
We added `Namespace()` as a method to `Blob` in celestia-core as a convenience. Perhaps you're missing this context: https://github.com/celestiaorg/celestia-core/pull/1030

We can create a separate issue to explore moving types like `Blob` and `BlobTx` to celestia-app but TBH I don't expect it to be possible without a large refactor.

Reply:
Aaah just found where it's needed, thanks :+1: